src/other/sample.jl

```diff
@@ -0,0 +1,28 @@
+import StatsBase: sample
+
+function sample(df::DataFrame)
+    sample(df, 1)
+end
+
+"""
+    sample(df[, N])
+
+Returns a (random) sample of a DataFrame
```

Comment:
I prefer something like *"Returns a (random) sample of rows from a DataFrame"* to be explicit about the selección of rows.